cli/config.go

```diff
@@ -8,6 +8,7 @@ import (
 	"os/signal"
 	"path/filepath"
 	"runtime"
+	"syscall"
```

Comment:
Use of syscall package in a multi-platform app might cause problems. You might want to create separate implementations for each platform or use generic signals I guess ?